### ISSUE TITLE
Nice Project

### ISSUE THREAD

**Issue by firmai** [pranjalphanse/Mahindra-First-Choice#1]
Nice project, but I can't find the data.

**Comment by pranjalphanse** [comment]
Yeah actually I can not upload dataset because it's illegal to share
companies dataset. I've uploaded PowerPoint presentation along with colab
notebook file.
